### ISSUE TITLE
Add player disconnect events to the output

### ISSUE THREAD

**Issue by dragunoff** [AMHOL/openra-ruby#13]
Outcome time and state is already extracted but time of disconnect could also be useful.

**Comment by AMHOL** [comment]
Not sure we can do this reliably as the replay comes from the client so we're not guaranteed a disconnect event for all clients

**Comment by dragunoff** [comment]
Yeah, it's always from the point of view of the client - so I think it's expected to not have a disconnect event from everyone.

**Comment by AMHOL** [comment]
OK, will look into adding this
